frontend/packages/console-app/src/redux/reducers/quick-start-reducer.ts

```diff
@@ -0,0 +1,31 @@
+import { QuickStartSidebarActions, Actions } from '../actions/quick-start-actions';
+import { RootState } from '@console/internal/redux';
+
+export const quickStartReducerName = 'quickStart';
+
+type State = {
+  isExpanded: boolean;
+  activeQSID: string;
```

Comment:
```suggestion
  activeQuickStartID: string;
```
We aren't abbreviating to `qs` anywhere else.

or even just:
```suggestion
  activeID: string;
```